modules/microshift-rhoai-servingruntimes-ex.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * microshift_ai/microshift-rhoai.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-rhoai-servingruntimes-ex_{context}"]
+= Creating a ServingRuntime CR for use in {microshift-short}
+
+Create a `ServingRuntime` custom resource (CR) based on installed manifests and release information. The included steps are an example of reusing the included `microshift-ai-model-serving` manifest files to re-create the {ovms} ({ov}) model-serving runtime in the workload namespace.
+
+[NOTE]
+====
+This approach does not require a live cluster, so it can be part of CI/CD automation.
+====
+
+.Prerequisites
+
+* The `microshift-ai-model-serving-release-info` RPM is installed.
```

Comment:
It also requires `microshift-ai-model-serving` which ships `/usr/lib/microshift/manifests.d/001-microshift-ai-model-serving/runtimes/ovms-kserve.yaml`.
Should we make it explicit or we assume user installed that RPM in earlier steps?

Reply:
Explicit. We can never really assume.